server/core/src/main/java/org/infinispan/server/core/transport/NettyChannelInitializer.java

```diff
@@ -43,7 +42,9 @@ public NettyChannelInitializer(ProtocolServer<A> server, NettyTransport transpor
    @Override
    public void initializeChannel(Channel ch) throws Exception {
    ChannelPipeline pipeline = ch.pipeline();
-      pipeline.addLast("stats", new StatsChannelHandler(transport));
+      if(transport != null) {
+         pipeline.addLast("stats", new StatsChannelHandler(transport));
```

Comment:
For now - yes. As it is now, the `StatsChannelHandler` requires `transport`. The multi-tenant router make it `null`, so we have a problem there. 

I'm planning to refactor it after this one gets in: https://issues.jboss.org/browse/ISPN-7057